lib/models/addon.js

```diff
@@ -412,16 +412,16 @@ var Addon = CoreObject.extend({
 
     @public
     @method included
-    @param {EmberApp} app The application object
+    @param {EmberApp|Addon} parent The parent object that included this addon
```

Comment:
I'm torn about the param type definition because of this: https://github.com/ember-cli/ember-cli/blob/master/lib/broccoli/ember-addon.js#L52-L56

Edit:
```diff
- that
+ which
```